cookbook/src/02-p2p.md

```diff
@@ -0,0 +1 @@
+# Receiving Data over P2P
```

Comment:
@tcharding any code snippet up your sleeve that I can deconstruct and explain in simple steps?

Reply:
None, I have not played around with p2p stuff very much at all.

Reply:
Ok I will remove it for now from the PR, we can revisit later